plugin/evm/mempool.go

```diff
@@ -152,6 +157,10 @@ func (m *Mempool) AddTx(tx *Tx) error {
 	defer m.lock.Unlock()
```

Comment:
addeed -> added? (couple lines above this)

Reply:
Re-spelled